test/spec/todo-item/todo-item.spec.ts

```diff
@@ -90,4 +90,63 @@ describe('POST /todos', () => {
         .to.equal('The title is empty or the title is not a string.')
     }
   })
+
+  describe('PUT /todos/:id', () => {
```

Comment:
We should have check for whether the API was successfully able to update the Todo when a valid request was submitted

Reply:
It is already there in the first test, it is checking if we have the id and title fields in the response, which says that the request was successfully submitted.